src/app/shared/styles/_typography.scss

```diff
@@ -0,0 +1,38 @@
+@mixin caption-1-normal {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 16px;
+}
+
+@mixin heading-1-medium {
+  font-size: 24px;
+  font-weight: 500;
+  line-height: 32px;
+}
+
+@mixin heading-2-medium {
+  font-family: Inter;
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 28px;
```

Comment:
usually  line-height should be in em

Reply:
@valeriya-reva 
It is hard to use. If design have only `px` I should every time transform `px` to `em` and if design change and I will review design where `line-heigth` in `px` and use `em` then I can miss change because I have `1.4em` and design have `27px`, but if I have `28px` then I can't miss this change.